app/models/tag.rb

```diff
@@ -23,6 +23,68 @@ class Tag < ActiveRecord::Base
   # the order is important, and it is the order in which they appear in the tag wrangling interface
   USER_DEFINED = ['Fandom', 'Character', 'Relationship', 'Freeform']
 
+  def is_this_a_large_tag(length_of_time_in_cache,how_long_it_takes_to_compute_size)
```

Comment:
For succinctness, maybe something more like `def is_large_tag(time_in_cache, time_to_compute_size)`?